app/src/main/resources/database/scripts/initial-schema.sql

```diff
@@ -6,4 +6,12 @@ CREATE TABLE recipe (
 	target_vg smallint NOT NULL,
 	base_nicotine smallint NOT NULL,
 	quantity smallint
+);
+
+CREATE TABLE recipe_flavourings (
+    recipe_id serial UNIQUE NOT NULL,
+    flavouring_id int NOT NULL PRIMARY KEY,
+    percentage decimal(2, 2) NULL,
+    FOREIGN KEY (recipe_id)
```

Comment:
It's also a good idea to have a more descriptive name for the foreign key, it's important to add some information to it.
Let's define the standard:
fk_foreignKeyTableName_PrimaryKeyTableName
So, in this case, it would be:
```fk_recipe_recipe_flavouring```